src/asciidoc/docs/develop/projects.adoc

```diff
@@ -1,156 +1,144 @@
 = Project structure and configuration
 :page-slug: projects
-:page-description: Description of the unique files and elements in a Koji project that power the magic of customization.
+:page-description: Description of the directory structure, files, and configuration data in a development project that power the features of a Koji app.
+:includespath: ../_includes
 
-Koji provides you with an environment for developing and publishing an application in your browser.
-The project environment includes a terminal connected to a running container with all the familiar tools, the same code editor that powers VSCode, and tools to help you deploy and publish your application to https://withkoji.com[withkoji.com].
+A repository on Koji contains everything that defines your development project, including the source code, assets, and configuration data.
+This topic describes a typical repository structure and defines the configuration data that is required to develop and deploy apps on Koji.
 
-== Project directory
+== Source control on Git
 
-The project directory at `/usr/src/app` includes a Git repository for your project with two remotes.
+The Koji platform uses Git repositories for source control.
+A directory on the Koji CDN stores the underlying files for each development project.
```

Comment:
This feels potentially confusing - maybe more "Your repository is hosted on Koji's CDN"?

Reply:
Awesome feedback! I made a change that I think addresses both comments. Let me know what you think!